changelogs/current.yaml

```diff
@@ -32,6 +32,13 @@ minor_behavior_changes:
 
 bug_fixes:
 # *Changes expected to improve the state of the world and are unlikely to have negative effects*
+- area: conn_pool
+  change: |
+    Fixed an issue that could lead to insuffient connections for current pending requests. If a connection starts draining while it
+    has negative unused capacity (which happens if a SETTINGS frame reduces allowed concurrency to below the current number of
+    requests), that connections unused capacity will be included in total pool capacity even though it is unusable because it is
```

Comment:
```suggestion
    Fixed an issue that could lead to insufficient connections for current pending requests. If a connection starts draining
    while it has negative unused capacity (which could happen if an HTTP/2 ``SETTINGS`` frame reduce allowed concurrency
    to below the current number of requests), that connections unused capacity will be included in total pool capacity even though it is unusable because it is
```